config/routes.rb

```diff
@@ -1,6 +1,4 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+ resources :users, only: [:index, :show]
+ resources :posts, only: [:index, :show]
```

Comment:
- [ ] It would be better if we match our routes with the wireframes. Hint: I would be better if we nest our post routes into the users routes.Like for example;
```
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
end

```

Reply:
thanks a lot, I already made the requested changes and will submit it again.